src/Hooks.php

```diff
@@ -59,11 +59,13 @@ public static function renderAudioButton( $input, array $args, $parser, $frame )
 			$mimetype = $file->getMimeType();
 
 			$vol = $args['vol'] ?? '1.0';
+			$preload = $args['preload'] ?? 'auto';
```

Comment:
I questioned in https://github.com/NilsEnevoldsen/AudioButton/issues/8 whether the default should be `preload="metadata"`. According to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio#preload,

> The default value is different for each browser. The spec advises it to be set to metadata.

Have you tested the implications of setting it to `auto` with different browsers? Do you know what the defaults are for different browsers? Is it possible we should leave off the entire property unless a preload argument is set, to allow the browser to determine the default?

Reply:
I'm also fine with default metadata then if not auto, or not having a default and only including the option to add a preload off. I am simply not sure how to leave this up to browser default while also adding the option, not that experienced in coding, I apologize.

The main intention of my commit is adding an _option_ to turn off preload, because already large pages (which a team I am part of are working with) that also include more than 1000 WebMediaPlayer objects on top of that size, are suffering from the 1001st & onward objects not loading at all and also very slow loading times for the 1000 first ones in general, which renders the pages and mediaplayers unusable and inaccessible.

Reply:
Yes, understood. @pcjtulsa any thoughts?

Reply:
Yes, I think it should still fallback to metadata if the parameter is not specified.

Reply:
To summarize.

Current behavior:

- Up to browser defaults
- Spec recommends browser defaults to `metadata`; in practice this may vary
- User cannot override

Proposed behavior, which we think is superior:

- Force `metadata`
- User can override
- Browser defaults are ignored

Correct?

Reply:
Yes, that sounds great! 🙂

Reply:
That's okay with me. Want to add a commit that does that to this PR?

Reply:
Done, let me know if anything is wrong.